torchquad/tests/simpson_test.py

```diff
@@ -89,6 +89,65 @@ def _run_simpson_tests(backend, _precision):
     for error in errors:
         assert error < 5e-9
 
+    if backend != "numpy":
```

Comment:
comment missing that the JIT section starts here (you had one in the others)

Reply:
done!